### ISSUE TITLE
Misleading warning "WARN[0000] No services to build"

### ISSUE THREAD

**Issue by mag37** [mag37/dockcheck#253]
I'm opening this issue to inform of an issue from upstream **Docker Compose**. It's not harmful, just a misleading warning that can cause some unnecessary headache or troubleshooting.

The message `WARN[0000] No services to build` is shown when recreating a container and messing up the "Running/up" -messaging somewhat.

Example:
```bash
Done pulling updates.
Recreating updated containers.

Now recreating (1/1): beszel
WARN[0000] No services to build                         
[+] up 1/1
 ✔ Container beszel Recreated    
```

This seems to be an unintended message that appeared since **Docker Compose V5+**. This message should probably only be shown when using the command `build` with docker compose and not the default behavior when just running `docker compose up ...`.

Hopefully this will be fixed upstream and rolled out soon.
